explorer/nodewatch/nodewatch/templates/base.html

```diff
@@ -70,7 +81,9 @@
         .then(response => response.json())
         .then(response => {
           const height = response['height'];
-          $('#nem-block-height').text(`${height} (Harlock)`);
+          const finalizedHeight = response['finalizedHeight'];
+          $('#nem-block-height').text(`${height}`);
+          $('#nem-finalization-height').text(`${finalizedHeight}`);
```

Comment:
What is the finalized height for NEM(current height - 360)?

Reply:
yes